sentences/sv/light_HassLightSet.yaml

```diff
@@ -10,3 +10,11 @@ intents:
           - "[sätt | vrid | dra [på] | skruva | dimmra] [upp | ner ] <alla> <ljuskällor> [<i_på>] <area>[et] [upp | ner ] till <brightness> [<ljusintensitet>]"
         slots:
           name: "all"
+
+      - sentences:
+          - "Färga <name> {color}"
```

Comment:
Maybe adding:
`
(färga | ändra | byt) <name> [till] {color}
`
To enable test:
`
ändra garagelampan till rött